### ISSUE TITLE
Cypress not able to run tests in Chrome version Version 84.0

### ISSUE THREAD

**Issue by malih2** [cypress-io/cypress#8420]
### Current behavior: 
When trying to run `.feature /.js` tests files using Chrome on a local server I see the error shown below:
```
Whoops, we can't run your tests.
This browser was not launched through Cypress. Tests cannot run.
```

I recently updated my chrome Enterprise version from 80 to 84. It was working earlier but not after the update. 
![image](https://user-images.githubusercontent.com/39099854/91351403-df4dd580-e7ad-11ea-9211-9197daacdd46.png)

### Desired behavior:
The test should be executed properly when running them on Chrome.

Open cypress: cypress open
Select a test in the user interface and run it (Chrome selected as the browser)
Notice that the tests don't get executed

### Test code to reproduce
Not relevant to the scenario. Same behaviour happen when running cypress examples.
`set DEBUG=cypress:*` and `cypress run` to see this error
added the log file here 
[cypress.log](https://github.com/cypress-io/cypress/files/5132673/cypress.log)

### Additional information
Currently, I am using Edge84 and it seems to be working. 

### Versions
Cypress version - 4.8.0
OS: Windows 64 bit
Browser : Chrome Enterprise 84

**Comment by jennifer-shehane** [comment]
Cypress uses a Chrome extension in order to run. Are you blocking specific Chrome extensions? Is so you'll need to unblock it following these instructions: https://on.cypress.io/troubleshooting#Allow-the-Cypress-Chrome-extension

**Comment by malih2** [comment]
I checked with our security team and chrome extensions weren't being blocked. I saw that some of the chrome issues were fixed in the latest 5.2 version. Tried that as well and still see the same error. 

I followed the steps mentioned here [https://docs.cypress.io/guides/references/troubleshooting.html#Isolate-the-Problem]  and this didn't seem to fix the issue. 

Additionally, I did run ‘cypress info’ command on the terminal to find out the browsers cypress can detect/run on my machine. Attached is the log
[cypressinfo.log](https://github.com/cypress-io/cypress/files/5276737/cypressinfo.log)


> Cypress uses a Chrome extension in order to run. Are you blocking specific Chrome extensions? Is so you'll need to unblock it following these instructions: https://on.cypress.io/troubleshooting#Allow-the-Cypress-Chrome-extension
